internal/cephfs/clone.go

```diff
@@ -155,21 +156,19 @@ func createCloneFromSnapshot(ctx context.Context, parentVolOpt, volOptions *volu
 	}
 	defer func() {
 		if err != nil {
-			var ecip ErrCloneInProgress
-			if !errors.As(err, &ecip) {
-				if dErr := purgeVolume(ctx, volumeID(vID.FsSubvolName), cr, volOptions, true); dErr != nil {
-					klog.Errorf(util.Log(ctx, "failed to delete volume %s: %v"), vID.FsSubvolName, dErr)
-				}
+			if dErr := purgeVolume(ctx, volumeID(vID.FsSubvolName), cr, volOptions, true); dErr != nil {
```

Comment:
why `ErrCloneInProgress` check is removed here?

Reply:
ah.. brought it back. thanks.